templates/confidential-computing/ai.html

```diff
@@ -0,0 +1,246 @@
+
+{% block content %}
+
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>Ubuntu for confidential&nbsp;AI</h1>
+      </div>
+      <div class="col p-section--shallow">
+        <p>Protect the confidentiality and integrity of your AI workloads at run-time across public and private clouds.</p>
+
+        <hr class="is-muted"/>
+        <a class="js-invoke-modal p-button--positive" href="/contact-us">Contact us</a>
+      </div>
+    </div>
+    <div class="u-fixed-width">
+      {{ image (
+      url="https://assets.ubuntu.com/v1/3ff12e71-Image.png",
+      alt="",
+      width="1232",
+      height="526",
+      hi_def=True,
+      loading="auto"
+      ) | safe
+      }}
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Why you need  Ubuntu for confidential AI</h2>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided u-no-margin--bottom">
+          <li class="p-list__item is-ticked">Harness AI's power with utmost security assurance</li>
+          <li class="p-list__item is-ticked">Protect your security-sensitive AI data at run-time</li>
+          <li class="p-list__item is-ticked">Fine-tune your model securely with your enterprise data</li>
+          <li class="p-list__item is-ticked">Safeguard your intellectual property and protect your model</li>
+          <li class="p-list__item is-ticked">Deploy your AI workload securely even in untrusted environments</li>
+          <li class="p-list__item is-ticked">Strengthen your compliance posture</li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>How confidential AI&nbsp;works</h2>
+      </div>
+      <div class="col p-section--shallow ">
+        <p>
+          Confidential AI is made possible thanks to confidential computing. Unlike traditional VMs, where you have to trust that the host software is also secure, confidential VMs only require you to trust the software running within it and the platform's hardware root of trust.
+        </p>
+        <div class="u-aspect-ratio--16-9 u-background--fade u-align--center u-vertically-center">
+          {{ image (
+          url="https://assets.ubuntu.com/v1/01006e0d-ai-diagram@2x.png",
+          alt="",
+          width="600",
+          height="338",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+          }}
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+
+        <hr class="p-rule is-fixed-width is-muted" />
+
+        <div class="row">
+
+          <div class="col-3 col-medium-2">
+
+            <h3 class="p-heading-icon__title p-heading--5">CPU-based confidential computing</h3>
```

Comment:
```suggestion
            <h3 class="p-heading--5">CPU-based confidential computing</h3>
```
This class doesn't seem to be doing anything here, is its usage intentional?